src/devices/rubicson_pool_48942.c

```diff
@@ -0,0 +1,108 @@
+/** @file
+    Rubicson pool thermometer 48942 decoder.
+
+    Copyright (C) 2022 Robert Högberg
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+*/
+
+/**
+The device uses OOK and fixed period PWM.
+- 0 is encoded as 240 us pulse and 480 us gap,
+- 1 is encoded as 480 us pulse and 240 us gap.
+
+A transmission consists of an initial preamble followed by sync
+pulses and the data. Sync pulses and data are sent twice.
+
+Preamble:
+     __      ____      ____      ____      ____
+    |  |____|    |____|    |____|    |____|    |__________
+    480 980  980  980  980  980  980  980  980  3880     [us]
+
+Sync pulses:
+     ___     ___     ___     ___
+    |   |___|   |___|   |___|   |___
+     730 730 730 730 730 730 730 730    [us]
```

Comment:
Is it important that this line is indented by 4 also? Imo it looks better together with the pulses above with 5 character indentation.

Is this markdown rendered somewhere?

Reply:
> Is it important that this line is indented by 4 also? Imo it looks better together with the pulses above with 5 character indentation.

Yes, four indents for a preformatted block. All indents are removed.

> Is this markdown rendered somewhere?

Yes ;) https://triq.org/explorer/

Reply:
Ok, I changed indentation for this line in the latest version.